File: C/Vetores/Soma_vetor.c
#include <stdio.h>




int main ()
{

    int N, i;

    printf ("Quantos numeros voce vai digitar? ");
    scanf ("%d", &N);

    double vet[N], media, soma;

    for (i = 0; i < N; i++)
    {
        printf ("Digite um numero: ");
        scanf ("%lf", &vet[i]);
    }
    printf ("\n");
    printf ("VALORES = ");
    for (i = 0; i < N; i++)
    {
        printf ("%.1lf  ", vet[i]);
    }

    soma = 0;
    for (i = 0; i < N; i++)
    {
        soma = soma + vet[i];
    }

    media = soma/N;

    printf ("\n");
    printf ("SOMA = %.2lf\n", soma);
    printf ("MEDIA = %.2lf\n", media);





    return 0;
}

File: Python/Vetores/Soma_vetor.py
N = int(input("Quantos numeros voce vai digitar? "))

vet = [0 for x in range(N)]

for i in range(0, N):
    vet[i] = float(input("Digite um numero: "))

print()
print("VALORES = ", end=" ")

for i in range(0, N):
    print(vet[i], end="  ")

soma = 0

for i in range(0, N):
    soma = soma + vet[i]

media = soma/N

print(f"\nSOMA = {soma:.2f}")
print(f"MEDIA = {media:.2f}")

